prow/jenkins/metrics.go

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
@smarterclayton @csrwng can you have a look in these metrics? We will be able to track response statuses and latencies with `jenkins_requests` and `jenkins_request_latency`.